File: CS/CustomSeriesPointDrawingSample/Model/Employee.cs
namespace CustomSeriesPointDrawingSample.Model {
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    public partial class Employee
    {
        [Key]
        [Required]
        [Column("EmployeeID")]
        public int EmployeeId { get; set; }

        [Required]
        [StringLength(20)]
        public string LastName { get; set; }

        [Required]
        [StringLength(10)]
        public string FirstName { get; set; }

        [Column(TypeName = "image")]
        public byte[] Photo { get; set; }

        public string FullName {
            get { return FirstName + " " + LastName; }
        }

        public virtual ICollection<Order> Orders { get; set; }
    }
}

File: VB/CustomSeriesPointDrawingSample/Model/Employee.vb
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace CustomSeriesPointDrawingSample.Model

    Public Partial Class Employee

        <Key>
        <Required>
        <Column("EmployeeID")>
        Public Property EmployeeId As Integer

        <Required>
        <StringLength(20)>
        Public Property LastName As String

        <Required>
        <StringLength(10)>
        Public Property FirstName As String

        <Column(TypeName:="image")>
        Public Property Photo As Byte()

        Public ReadOnly Property FullName As String
            Get
                Return FirstName & " " & LastName
            End Get
        End Property

        Public Overridable Property Orders As ICollection(Of Order)
    End Class
End Namespace
